API.md

```diff
@@ -499,3 +499,72 @@ MixPanel](https://permanent.atlassian.net/wiki/spaces/EN/pages/2086862849/Mixpan
 ```
 {}
 ```
+
+
+## FusionAuth Users and Two-Factor Methods
+
+(Note: this section is not yet implemented)
+
+### GET `/idpuser`
+
+The logged-in user should only be able to get information about
+themselves, so no need to pass the email. We will use the one in the
+JWT.
+
+- Headers: Authorization: Bearer \<JWT from FusionAuth>
+- Response: For now, this endpoint will only return the user's
+  two-factor methods. This will be an array with 0-2 methods in it.
+
+```
+[
+  {
+    methodId: [string] 4 random letters and numbers
```

Comment:
The brackets around here suggest that the type of `methodId` is an array of strings, but I suspect it's just a string.